internal/server/asset.go

```diff
@@ -2,7 +2,7 @@ package server
 
 import (
 	"bytes"
-	"crypto/sha1" //nolint:gosec // Usage is for collision avoidance not security.
+	"crypto/sha1" //
```

Comment:
```suggestion
	"crypto/sha1" //nolint:gosec // Usage is for collision avoidance not security.
```

This needs to be restored.